sample_client/build.gradle

```diff
@@ -11,7 +11,7 @@
 
 buildscript {
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.7.2'
+        classpath 'com.android.tools.build:gradle:8.7.3'
```

Comment:
Also here. Please split it into another pr.

Reply:
Same here. I have to test the client app, otherwise it's not compiling.